docs/source/quickstart.rst

```diff
@@ -59,6 +59,32 @@ without any arguments::
 To save the logcat output into a file named `my_log.txt` (the file will appear in your current directory)::
 
     buildozer -v android debug deploy run logcat > my_log.txt
+    
+To see your running application's print() messages and python's error messages, use:
+
+::
+
+    buildozer -v android deploy run logcat | grep python
+
+
+Run my application on Windows 10
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Plug your Android device on a USB port
+
+- Open Windows PowerShell, go into the folder where you installed the Windows version of ADB, and activate the adb daemon:
+
+::
+
+    cd C:\platform-tools\
+    .\adb.exe devices
```

Comment:
Why not using buildozer here as well?
```
buildozer android adb -- devices
```

Reply:
https://github.com/kivy/buildozer/pull/1123#discussion_r433001557